src/StrawberryfieldFilePersisterService.php

```diff
@@ -191,39 +193,51 @@ public function getDestinationUri(
     // If no destination scheme was setup on our global config use the original file scheme.
     $desired_scheme = !empty($this->destinationScheme) ? $this->destinationScheme : $file_parts['destination_scheme'];
 
-    $destination_filename = $file_parts['destination_filetype'] . '-' . $file_parts['destination_filename'] . '.' . $file_parts['destination_extension'];
+    // Mime type becomes suffix. Performant for filtering in S3.
```

Comment:
@DiegoPino I think the comment is wrong (code is OK): 
// Mime type becomes prefix. UUID becomes suffix. Performant for filtering in S3.